components/prize_pool/commons/prize_pool_import.lua

```diff
@@ -144,10 +144,10 @@ function Import._computeGroupTablePlacementEntries(standingRecords, options)
 				hasOvertime = record.hasOvertime,
 			}
 
-			if not record.extradata.placeRange then
+			if not record.extradata.placeRange and record.extradata.finished then
```

Comment:
Could be a good idea to not assume datatype of anything in extradata, so unless it's already been parsed, `readBool`?

Reply:
```suggestion
			if not record.extradata.placeRange and Logic.readBool(record.extradata.finished) then
```